### PR TITLE
Since February 2021, Sonatype requires using the new s01.oss.sonatype.org host

### PR DESCRIPTION
**Pull request by csaltos**
Just a quick note added to documentation for using Sonatype with the new account servers

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @csaltos,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

File: src/reference/01-General-Info/05-Using-Sonatype.md [hunk 1]
```diff
@@ -244,6 +244,9 @@ range (e.g. Umlauts). If you are absolutely sure that you typed the
 right phrase and the error doesn't disappear, try changing the
 passphrase.
 
+> *Note:* If you are using a new OSSRH account created after February 2021,
+ please use `"s01.oss.sonatype.org"` instead of `"oss.sonatype.org"`
```
Comment (eed3si9n) [inline src/reference/01-General-Info/05-Using-Sonatype.md]: nit: could we remove "please" here for consistency plz?

Comment (csaltos) [reply src/reference/01-General-Info/05-Using-Sonatype.md]: sure !!

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**